### PR TITLE
Fixes issue #140

### PR DESCRIPTION
**Pull request by Nowheresly**
- fixes javadoc
- add unit test

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Nowheresly__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: src/test/java/microsoft/exchange/webservices/data/UniqueBodyTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
+package microsoft.exchange.webservices.data;
```
Comment (vboctor) [inline src/test/java/microsoft/exchange/webservices/data/UniqueBodyTest.java]: Add file header.

File: src/test/java/microsoft/exchange/webservices/data/UniqueBodyTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  final String text = "test";
+
+  final BodyType bodyType = BodyType.HTML;
+
+  @Mock EwsServiceXmlReader reader;
+  @Mock EwsServiceXmlWriter writer;
+
+
+  @Before public void setUp() throws Exception {
+
```
Comment (vboctor) [inline src/test/java/microsoft/exchange/webservices/data/UniqueBodyTest.java]: There seems to be randomly blank lines at the beginning or end of functions.  There also sometimes double blank lines.  I would be consistent with no blank lines at top or bottom of functions and removing doubles blank lines (make them single).

File: src/test/java/microsoft/exchange/webservices/data/UniqueBodyTest.java (hunk 1 above)
Comment (Nowheresly) [inline src/test/java/microsoft/exchange/webservices/data/UniqueBodyTest.java]: Oh my gosh!
You're right!

**Comment by serious6** [comment]
@Nowheresly maybe you can update this pr to the head revision and maybe you can also add javadoc  comments for the added tests? thanks in advance

**Comment by Nowheresly** [comment]
Sure I can.
Work in progress...

**Comment by Nowheresly** [comment]
done

**Comment by serious6** [comment]
@Nowheresly thanks for your contribution

**Pull request merged**